testsuite/features/support/commonlib.rb

```diff
@@ -228,3 +228,16 @@ def generate_repository_name(repo_url)
   repo_name.delete_prefix! 'http://minima-mirror-qam.mgr.prv.suse.net/ibs/SUSE:/Maintenance:/'
   repo_name
 end
+
+def extract_logs_from_node(node)
+  _os_version, os_family = get_os_version(node)
+  if os_family =~ /^opensuse/
+    node.run('zypper mr --enable os_pool_repo os_update_repo') unless $qam_test
+    node.run('zypper --non-interactive install tar')
+    node.run('zypper mr --disable os_pool_repo os_update_repo') unless $qam_test
+  end
+  node.run("journalctl > /var/log/messages && (tar cfvJP /tmp/#{node.full_hostname}-logs.tar.xz /var/log/ || [[ $? -eq 1 ]])")
```

Comment:
This might fail in SLES11 nodes, right? :thinking:

Reply:
No idea.